services/cases-api/lambdas/create.js

```diff
@@ -1,88 +1,69 @@
 import to from 'await-to-js';
 import uuid from 'uuid';
+import * as dynamoose from 'dynamoose';
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
 
-import { validateEventBody } from '../../../libs/validateEventBody';
-import * as response from '../../../libs/response';
-import { validateKeys } from '../../../libs/validateKeys';
 import config from '../../../config';
-import { CASE_ITEM_TYPE } from '../helpers/constants';
+import { CASE_ITEM_TYPE as ITEM_TYPE } from '../helpers/constants';
 import { decodeToken } from '../../../libs/token';
-// todo: move to libs as it's used by forms too
-import { putItem } from '../helpers/queries';
+import * as response from '../../../libs/response';
+
+import casesSchema from '../schema/casesSchema';
+
+const ddb = new dynamoose.aws.sdk.DynamoDB();
+dynamoose.aws.ddb.set(ddb);
+
+const Cases = dynamoose.model(`${config.resourcesStage}-${config.cases.tableName}`, casesSchema, {
+  create: false,
+});
 
 /**
- * Handler function for creating a case and store in dynamodb
+ * Handler create and store cases into AWS DynamoDB
```

Comment:
Suggest a combination of the two ;)
Handler function for creating and storing cases into AWS DynamoDB